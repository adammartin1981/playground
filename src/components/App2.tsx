import * as React from 'react'

export const App2 = () => <>Is this gonna fail</>
