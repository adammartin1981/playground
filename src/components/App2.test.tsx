import * as React from 'react'
import { App2 } from './App2'
import { shallow, configure } from 'enzyme'
import * as Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })

const app2 = shallow(<App2 />)

describe('App2', () => {
  it('should do the snapshot', () => {
    expect(app2).toMatchSnapshot()
  })
})
