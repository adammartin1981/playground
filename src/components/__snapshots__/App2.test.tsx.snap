// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App2 should do the snapshot 1`] = `
<Fragment>
  Foo
</Fragment>
`;
